tests/backup/backup_psa_test.go

```diff
@@ -943,3 +946,606 @@ var _ = Describe("{RestoreFromHigherPrivilegedNamespaceToLower}", Label(TestCase
+		credUid                                                               string
+		srcClusterUid                                                         string
+		clusterStatus                                                         api.ClusterInfo_StatusInfo_Status
+		preRuleName                                                           string
+		postRuleName                                                          string
+		preRuleUid                                                            string
+		postRuleUid                                                           string
+		customRestoreWithNamespaceAndStorageClassMapping                      string
+		customRestoreWithNamespaceAndStorageClassMappingToPrivilegedNamespace string
+		customRestoreWithNamespaceAndStorageClassMappingToBaselineNamespace   string
+		defaultRestoreName                                                    string
+		singleNSbackupNameWithVolumeAndResourceWithBaselineLabel              string
+		singleNSBackupNameWithVolumeAndResourceWithRestrictedLabel            string
+		multiNSBackupNameWithVolumeAndResourceWithRestrictedLabel             string
+		multiNSBackupNameWithVolumeAndResourceWithBaselineLabel               string
+		backupNameWithMultipleNsHavingDiffPSALabel                            string
+		periodicSchedulePolicyName                                            string
+		periodicSchedulePolicyUid                                             string
+		baselineNamespaceInSrcCluster                                         string
+		restrictedNamespaceInSrcCluster                                       string
+		privilegedNamespaceInSrcCluster                                       string
+		appNamespaces                                                         []string
+		backupList                                                            []string
+		restoreList                                                           []string
+		defaultAppList                                                        []string
+		scheduleNames                                                         = make([]string, 0)
+
+		sourceScNameList               []*storageApi.StorageClass
+		scheduledAppContexts           []*scheduler.Context
+		baselineScheduledAppContexts   []*scheduler.Context
+		restrictedScheduledAppContexts []*scheduler.Context
+		privilegedScheduledAppContexts []*scheduler.Context
+		allScheduledAppContexts        []*scheduler.Context
+		namespaces                     []string
+		psaApp                         []string
+		backupNamespaceMap             = make(map[string]string)
+	)
+
+	backupLocationMap := make(map[string]string)
+	numberOfBackups, _ := strconv.Atoi(GetEnv(MaxBackupsToBeCreated, "3"))
+	params := make(map[string]string)
+	storageClassMapping := make(map[string]string)
+	ctx, err := backup.GetAdminCtxFromSecret()
+	log.FailOnError(err, "Getting admin context from secret")
+
+	JustBeforeEach(func() {
+		defaultAppList = Inst().AppList
```

Comment:
Please call the `StartPxBackupTorpedoTest` function. This is mandatory for all tests.